recipe/build.sh

```diff
@@ -11,19 +11,27 @@ for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
     [[ $i == "conda_build.sh" ]] && continue
     [[ $i == "metadata_conda_debug.yaml" ]] && continue
-    if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+    if [[ $i == "bin" ]]; then
+        mkdir -p ${PREFIX}/${targetsDir}
+        mkdir -p ${PREFIX}/$i
+        cp -rv $i ${PREFIX}/${targetsDir}
+        cp ${PREFIX}/${targetsDir}/bin/nvcc.profile ${PREFIX}/bin/nvcc.profile # need to copy so we can patch
```

Comment:
Very minor nit, regarding the comment: I think at this point the file has been patched and we're copying the patched file to the right place, so technically "copy the patched file" is more accurate than "need to copy so we can patch".

Reply:
This has been dropped in the latest version, as we need separate `nvcc.profile` files in `$PREFIX/bin` and `$PREFIX/targets/<arch>/bin`.